.babelrc

```diff
@@ -1,7 +1,12 @@
 {
```

Comment:
Pls update [CHANGELOG.md](https://github.com/amazon-connect/amazon-connect-chatjs/blob/master/CHANGELOG.md) with details of changes.

Reply:
Also need to update version in https://github.com/amazon-connect/amazon-connect-chatjs/blob/master/package.json#L3

Reply:
Also, can you mention in the ChaangeLog file - how can Customers disable CSM tracking.